### PR TITLE
Bug 1910036: Gherkin Script Scenario for epic ODC-4504

### PR DESCRIPTION
**Pull request by gajanan-more**
Fixes: https://issues.redhat.com/browse/ODC-4847

Problem: Gherkin Scripts for scenarios for [ODC-4504](https://issues.redhat.com/browse/ODC-4504)

Solution: These are the gherkin scripts designed covering all the scenarios related to epics [ODC-4504](https://issues.redhat.com/browse/ODC-4504)

### CODE REVIEW

**Review by reginapizza — APPROVED** [review]

**Review by chetan-rns — APPROVED** [review]
/lgtm

File: frontend/packages/dev-console/integration-tests/features/gitops/linkArgoCD.feature [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        And user has created SealedSecretController CR
+        And user has installed OpenShift Pipelines Operator
+        And user has installed Argo CD Operator
+        And user has installed GitOps Operator
+
+    
+    @regression, @manual
+    Scenario: External link to the ArgoCD Dashboard
+        Given user has created namespace "test-argocd"
+        And user has logged in to cluster using CLI
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/gitops/linkArgoCD.feature:17]: Cant we login using web terminal operator

File: frontend/packages/dev-console/integration-tests/features/gitops/linkArgoCD.feature [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        And user has imported a gitops repo using kam command
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/gitops/linkArgoCD.feature:18]: Can you please add test data

File: frontend/packages/dev-console/integration-tests/features/gitops/linkArgoCD.feature (hunk 1 above)
Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/features/gitops/linkArgoCD.feature:17]: We can do it but for that we have to add it in the background. Should I do it?

**Pull request merged**